tensorflow_datasets/scripts/random_image_generator.py

```diff
@@ -0,0 +1,132 @@
+
```

Comment:
Thank you for the update. Could you format your code correctly with 
https://www.tensorflow.org/datasets/add_dataset#5_check_your_code_style

Reply:
I already format the code using pylint and that shows 10/10 results.

Reply:
We are not using the standard pytype, but the Google python style guide. Please use instructions at: https://www.tensorflow.org/datasets/add_dataset#5_check_your_code_style

Reply:
Sure

Reply:
So i need to use lint.sh in oss_scripts.

Reply:
Yes please, as explained in https://www.tensorflow.org/datasets/add_dataset#5_check_your_code_style

Reply:
could you please help  me with lint.sh i got error while running this script like unexpected end of file this is just the last step that remains. Please help me with this i am using windows.
Thank you

Reply:
Without knowing which error you're getting it's difficult to help. Can you copy the full message you're getting ?

Reply:
lint.sh: line 2: $'\r': command not found
lint.sh: line 4: $'\r': command not found
lint.sh: line 11: syntax error: unexpected end of file

Reply:
The script may not be available on windows.

You'll have to manually install pylint and use it with this config file: https://raw.githubusercontent.com/tensorflow/tensorflow/master/tensorflow/tools/ci_build/pylintrc